src/css/skins/six.less

```diff
@@ -114,11 +119,12 @@
         }
 
         ::-webkit-scrollbar {
-            border-radius: .25em;
+            border-radius: .25em !important;
```

Comment:
We try not to use ```!important``` - is there a way to not use important and achieve the same goal?